app/steps/ui/summary/index.js

```diff
@@ -9,7 +9,7 @@ const services = require('app/components/services');
 const ExecutorsWrapper = require('app/wrappers/Executors');
 const WillWrapper = require('app/wrappers/Will');
 const FormatName = require('app/utils/FormatName');
-const featureToggle = require('app/utils/FeatureToggle');
+const FeatureToggle = require('app/utils/FeatureToggle');
```

Comment:
It would be more consistent with the code base for this to begin with a lowercase letter. Uppercase has been generally used to instantiate classes.

So `featureToggle` rather than `FeatureToggle`.